### PR TITLE
Add unicode latex and html entity fix functions

### PR DESCRIPTION
**Pull request by WSchaatsbergen**
* Added four functions to `TextScrubber`
* Upgraded Sphinx because the old version didn't run anymore
* Added Python version 3.10, 3.11 and 3.12 to the builds
* Removed support for 3.6 because it doesn't have str.isascii, and it's long past its support date
* Set version to 0.5.0

### CODE REVIEW

File: .github/workflows/github-pages.yml [hunk 1]
```diff
@@ -8,11 +8,11 @@ jobs:
...
 
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v4
     - name: Set up Python
-      uses: actions/setup-python@v2
+      uses: actions/setup-python@v5
       with:
-        python-version: '3.6'
+        python-version: '3.12'
```
Comment (sybrenjansen) [inline .github/workflows/github-pages.yml]: I have a feeling the sphinx-versioning thing won't work under python 3.12. Have you tested it locally?

I still have to fix the whole versioning thing and do it differently, same for mpire. But didn't take the time yet to fix that. If it's not working locally, it's best to revert to 3.6 for the docs for now

File: text_scrubber/text_scrubber.py [hunk 2]
```diff
@@ -187,6 +231,17 @@ def lowercase(self, on_tokens: bool = False, name: str = 'lowercase') -> 'TextSc
         """
         return self._add_step(name, str.lower, on_tokens)
 
+    def normalize_unicode(self, form: str, on_tokens: bool = False, name: str = 'normalize_unicode') -> 'TextScrubber':
```
Comment (sybrenjansen) [inline text_scrubber/text_scrubber.py]: If there's a limited number of valid values, consider using `Literal["NFC", "NFKC", "NFD", "NFKD"]` as type

**Review by sybrenjansen — CHANGES_REQUESTED** [review]
I will create a small PR with levenshtein changes. Would be great if we can merge that one before this one, so we only have one version release.

The changes look good though. Mainly putting it to 'request changes' to avoid merging it before the other one.

File: .github/workflows/python-package.yml [hunk 3]
```diff
@@ -15,17 +15,17 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
```
Comment (sybrenjansen) [inline .github/workflows/python-package.yml]: 3.7 is ook EOL

**Review by sybrenjansen — CHANGES_REQUESTED** [review]

File: .github/workflows/github-pages.yml (hunk 1 above)
Comment (WSchaatsbergen) [reply .github/workflows/github-pages.yml]: The documentation did build on 3.12 without errors. But I can revert back to 3.6 if you like. Or shall I do 3.8 as that's now the lowest version TextScrubber supports?

**Review by sybrenjansen — DISMISSED** [review]

**Review by sybrenjansen — APPROVED** [review]

**Pull request merged**